docs/loader-api.md

```diff
@@ -141,3 +141,60 @@ require("~/lib/utils")
 // es6
 import * as utils from "~/lib/utils";
 ```
+
+## Loader Plugins
+Loader plugins can intercept certain default behaviors. Here is the current plugin interface: 
+
+```js
+interface LoaderPlugin {
+    /** 
+     * If true is returned by the plugin
+     *  it means that module change has been handled
+     *  by plugin and no special work is needed by FuseBox
+     **/
+    hmrUpdate?(evt: SourceChangedEvent): boolean;
+}
+
+/** Where */
+type SourceChangedEvent = {
```

Comment:
You've changed `type` to `interface` in `LoaderAPI.ts` - I'd propose to change this line, too. 😃